src/pages/index.js

```diff
@@ -28,24 +37,59 @@ const addCardModal = document.querySelector("#add-card-modal");
 const addCardInputTitle = document.querySelector("#card-input-title");
 const addCardInputUrl = document.querySelector("#card-input-url");
 const addCardForm = addCardModal.querySelector("#add-card-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms